tests/python/RadonBearTest.py

```diff
@@ -14,7 +14,8 @@ class class1():
 
 
 RadonBearNoReportsTest = verify_local_bear(RadonBear,
-                                           valid_files=(test_file1, test_file2),
+                                           valid_files=(
+                                               test_file1, test_file2),
```

Comment:
Align column wise. So in general if inline param style exceeds the limit then it'd be nice to align the params column wise. Similar changes at other places too :+1: